### DISCUSSION TITLE
failed to start vs code remote server.

### DISCUSSION THREAD

**Discussion by sridbulli** [community/community#132741] [GitHub Education]
### Hi everyone,

An unexpected error occurred that requires a reload of this page.
The workbench failed to connect to the server (Error: exception was thrown by handler. exception: failed to start vs code remote server.)

**Comment by sridbulli** [comment]
plz help its codespace

**Reply by PyPilot-ai** [reply]
Same deal.

**Reply by PyPilot-ai** [reply]
> plz help its codespace

There is a downage. 

https://www.githubstatus.com/

"Update - We are investigating issues with Codespaces. Some users may not be able to connect to their Codespaces at this time. We will update you on mitigation progress.
Jul 19, 2024 - 02:43 UTC
Investigating - We are investigating reports of degraded performance for Codespaces
Jul 19, 2024 - 02:10 UTC"

**Comment by sridbulli** [comment]
now working for me
